app/models/source.js

```diff
@@ -22,12 +24,28 @@ var methods = Object.assign({
     });
   },
 
-  itemsGetUrl: function(properties) {
-    if (properties.next) {
-      return properties.next;
+  itemsGetUrl: function(itemsGetUrlTemplate, $properties) {
+
+    if ($properties.next) {
+      return $properties.next;
     }
 
-    return templateCompiler(this.itemsGetUrlTemplate, properties);
+    return templateCompiler(itemsGetUrlTemplate, $properties);
+  },
+
+  /**
+   * @param done {Function} A callback that will be passed a the error (Error) and optionally the sourceContentTypes results
+   */
+  getSourceContentTypesForSource: function(done) {
+    SourceContentType.find({source: this.id}, function(err, sourceContentTypes) {
+      if (err) {
+        return done(err);
+      } else {
+
```

Comment:
I think this rule should be able to prevent these extra line breaks as well: https://eslint.org/docs/rules/padding-line-between-statements